pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/PersistentTopics.java

```diff
@@ -810,6 +810,11 @@ public void updatePartitionedTopic(
             @ApiParam(value = "The number of partitions for the topic",
                     required = true, type = "int", defaultValue = "0")
                     int numPartitions) {
+        validateTopicName(tenant, namespace, encodedTopic);
```

Comment:
I'm actually not following here.
You add a call to `validateTopicName`, but in line 818, you have a call to `validatePartitionedTopicName` which is:

```
        // first, it has to be a validate topic name
        validateTopicName(tenant, namespace, encodedTopic);
        // second, "-partition-" is not allowed
        if (encodedTopic.contains(TopicName.PARTITIONED_TOPIC_SUFFIX)) {
            throw new RestException(Status.PRECONDITION_FAILED,
                    "Partitioned Topic Name should not contain '-partition-'");
        }
```

This seems to contain exactly the code you've added. So wham I missing here?

Reply:
Sorry, I forget to delete line818 and has been fixed it.